internal/provider/resource_application_test.go

```diff
@@ -580,17 +607,21 @@ resource "juju_application" "{{.AppName}}" {
     channel  = "latest/edge"
   }
 
-  {{ if ne .ConfigParamName "" }}
+  {{- if ne .ConfigParamName "" }}
   config = {
     {{.ConfigParamName}} = "{{.ConfigParamName}}-value"
   }
-  {{ end }}
+  storage = [{
+    label = "runner"
+    size = "107G"
+  }]
+  {{- end }}
 
-  {{ if ne .ResourceParamName "" }}
+  {{- if ne .ResourceParamName "" }}
   resources = {
     {{.ResourceParamName}} = {{.ResourceParamRevision}}
   }
-  {{ end }}
+  {{- end }}
```

Comment:
This change is having unintended consequences in other tests I think, I see new failures in TestAcc_ResourceOffer and TestAcc_ResourceIntegrationWithMultipleConsumers.

Reply:
That strange, because these changes only remove empty lines from the result plan.
But OK I can revert them.